Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -143,15 +149,26 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		case R.id.action_add_claim:
 			startAddExpenseClaimActivity();
 			return true;
+		case R.id.action_sort_claim:
+			startSortExpenseClaimActivity();
+			return true;
 		default:
 			return super.onOptionsItemSelected(item);
 		}
 	}
+
 	
 	private void startAddExpenseClaimActivity() {
 		Intent addIntent = new Intent(this, ExpenseClaimAddActivity.class);
 		startActivityForResult(addIntent, ADD_EXPENSE_CLAIM_REQUEST);
 	}
+	
+	
+	private void startSortExpenseClaimActivity() {
+		// Toast.makeText(this,  "Sort Claim", Toast.LENGTH_SHORT).show();
+		Intent intent = new Intent(this, ExpenseClaimSortActivity.class);
+		startActivity(intent);
```

Comment:
Related to above comments, use `startActivityForResult()` instead (like in `startAddExpenseClaimActivity()`) so that you can extract the sorted array from the resulting intent.